libs/api/repository/src/lib/gn4/auth/auth.service.ts

```diff
@@ -19,6 +20,7 @@ export const LOGIN_URL = new InjectionToken<string>('loginUrl')
 export class AuthService {
   authReady$: Observable<UserModel>
   user$: Observable<UserModel>
+  allUsers$: Observable<any[]>
```

Comment:
Do you think you can make this be `Observable<UserModel[]>` instead? `any` should really be avoided, it means everything downstream will not be correctly typechecked.

Reply:
Sure, can do :slightly_smiling_face: